pkgs/development/compilers/dmd/generic.nix

```diff
@@ -209,6 +211,12 @@ stdenv.mkDerivation rec {
     runHook postInstall
   '';
 
+  preFixup = ''
+    find $out/bin -type f -exec ${removeReferencesTo}/bin/remove-references-to -t ${HOST_DMD} '{}' +
```

Comment:
You should change `HOST_DMD` to `dmdBootstrap` here.

Reply:
Good catch